yt/frontends/ramses/particle_handlers.py

```diff
@@ -54,7 +57,7 @@ class ParticleFileHandler(object):
     local_particle_count = None  # The number of particle in the domain
 
 
-    def __init__(self, ds, domain_id):
+    def __init__(self, ds, domain):
```

Comment:
Uh oh, and just when I thought I fully grokked the `domain_id` / `domain` distinction... :smile: